src/Resources/public/administration/css/adyen-payment-shopware6.css

```diff
@@ -1 +1,4 @@
-.adyen-card-header{display:flex;height:30px;margin-bottom:15px}.adyen-card-header-first{margin-top:auto;width:100%}
\ No newline at end of file
+.adyen-card-header{display:flex;height:30px;margin-bottom:15px}.adyen-card-header-first{margin-top:auto;width:100%}
```

Comment:
Why do we have duplicated line of code?

Reply:
This is due to the wrong implementation of SCSS in the plugin and we don't have direct control over this file since it's a generated asset. It will be a follow-up since it's not in the scope of Vuejs 3 migration.